samples/Integration/CMake-fetch-content-app/README.md

```diff
@@ -0,0 +1,42 @@
+# Integrating the Azure SDK for C++ with CMake fetch content to your application
+
+This application shows how to integrate the Azure SDK for C++ in your application. It uses CMake fetch content functionality to clone the Azure SDK for C++ repo into your `build` directory. This approach is useful when using CMake to build your application.
+
+## Pre-requisites
+
+Install the [Azure SDK for C++ dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#third-party-dependencies).
+
+- CMake project (min version 3.13).
+- C++ version 14 or greater.
+
+## Build
+
+```bash
+#
+# Building the application.
+# Instructions from application root directory.
+#
+
+# Create build directory just the first time.
+mkdir build
+cd build
+
+# Generate and build
+# This code assumes that the SDK dependencies were installed with VCPKG
+cmake -DCMAKE_TOOLCHAIN_FILE=path/to/vcpkg/scripts/buildsystems/vcpkg.cmake ..
+cmake -build .
+```
+
+## Run application
+
+Review source code header for `environment variables` that must be set up before running the app.
+
+```bash
+#
+# Running the Application
+# Instructions from inside the build directory.
+#
+
+# Run binary (.exe on Windows)
+./applicationName
```

Comment:
Is this the name of the exe that gets built? Where is this defined? Is it not `application`?

Reply:
Tt is `application`, yes, 
it is defined in the CMake file with the `add_executable` instruction.

I used this name to avoid a hard relationship between the Readme and the code.  Then I the name in the sample can change in the future and this is still valid

Reply:
I don't think having a hard relationship is much of a concern here since we shouldn't really be changing the samples all that much.

The way it is written now is more confusing to follow along (especially since there is no indication where this is defined or what the name ought to be).

Making it straight-forward to follow along and get the application running, without any changes, is valuable for adoption.
```suggestion
./application
```